<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopicWindow</class>
 <widget class="QWidget" name="TopicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Topic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="publishLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="publishButton">
       <property name="text">
        <string>Publish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="subscribeCheckBox">
       <property name="text">
        <string>Subscribe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="subscribeTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
